codecoverage.py

```diff
@@ -71,7 +77,8 @@ def download_coverage_artifacts(build_task_id):
         if 'target.code-coverage-gcno.zip' in artifact['name']:
             download_artifact(build_task_id, artifact)
 
-    test_tasks = [t for t in get_tasks_in_group(task_data['taskGroupId']) if t['task']['metadata']['name'].startswith('test-linux64-ccov')]
+    test_tasks = [t for t in get_tasks_in_group(task_data['taskGroupId']) if t['task']['metadata']['name'].startswith('test-linux64-ccov') and (suites is None or suite_name_from_task_name(t['task']['metadata']['name']) in suites)]
```

Comment:
Also, suggestion: It would be helpful to print a warning if a requested suite is not available.